docs_user/modules/proc_adopting-autoscaling.adoc

```diff
@@ -20,48 +20,60 @@ should be already adopted.
 . Patch the `OpenStackControlPlane` CR to deploy autoscaling services:
 +
 ----
-cat << EOF > aodh_patch.yaml
+oc patch openstackcontrolplane openstack --type=merge --patch '
 spec:
-  autoscaling:
-    enabled: true
-    prometheus:
-      deployPrometheus: false
-    aodh:
-      customServiceConfig: |
-        [DEFAULT]
-        debug=true
-      secret: osp-secret
-ifeval::["{build}" != "downstream"]
-      apiImage: "quay.io/podified-antelope-centos9/openstack-aodh-api:current-podified"
-      evaluatorImage: "quay.io/podified-antelope-centos9/openstack-aodh-evaluator:current-podified"
-      notifierImage: "quay.io/podified-antelope-centos9/openstack-aodh-notifier:current-podified"
-      listenerImage: "quay.io/podified-antelope-centos9/openstack-aodh-listener:current-podified"
-endif::[]
-ifeval::["{build}" == "downstream"]
-      apiImage: "registry.redhat.io/rhosp-dev-preview/openstack-aodh-api-rhel9:18.0"
-      evaluatorImage: "registry.redhat.io/rhosp-dev-preview/openstack-aodh-evaluator-rhel9:18.0"
-      notifierImage: "registry.redhat.io/rhosp-dev-preview/openstack-aodh-notifier-rhel9:18.0"
-      listenerImage: "registry.redhat.io/rhosp-dev-preview/openstack-aodh-listener-rhel9:18.0"
-endif::[]
-      passwordSelectors:
-      databaseUser: aodh
-      databaseInstance: openstack
-      memcachedInstance: memcached
+  telemetry:
+    template:
+      autoscaling:
+        aodh:
+          databaseAccount: aodh
+          databaseInstance: openstack
+          passwordSelector:
+            aodhService: AodhPassword
+            ceilometerService: CeilometerPassword
+          preserveJobs: false
+          rabbitMqClusterName: rabbitmq
+          secret: osp-secret
+          serviceUser: aodh
+        enabled: true
+        heatInstance: heat
 EOF
 ----
 
-. Optional: If you have previously backed up your {OpenStackShort} services configuration file from the old environment, you can use os-diff to compare and make sure the configuration is correct. This will producre the difference between both ini configuration files:
+. Patch the `OpenStackControlPlane` CR to deploy cluster-observability-operator:
```

Comment:
COO should be installed before patching the control plane.

MetricsStorage will fail dues to a missing dependency otherwise